docs/tutorial/file.rst

```diff
@@ -166,6 +166,21 @@ Plotting is typically performed with the origin of the plot set to the timestamp
     first_slice.plot()
     second_slice.plot(start=first_slice.start)  # we want to use the start of first_slice as time point "zero"
 
+Masking
+^^^^^^^
+
+Similarly, a channel can be masked using a boolean mask.
+For example, forces above 5 pN can be excluded as follows::
+
+    mask = file.force1x.data <= 5
+    masked = file.force1x[mask]
+
+Multiple criteria can be combined by using `numpy`'s logic operators.
+For example, restricting the forces between 2 and 5 can be accomplished as follows::
+
+    mask = np.logical_and(file.force1x.data > 2, file.force1x.data < 5)
+    masked = file.force1x[mask]
+
```

Comment:
It would be good to follow `numpy` naming conventions here and call this boolean indexing: https://numpy.org/doc/stable/user/basics.indexing.html#boolean-array-indexing This is because masks and masked arrays have a very different meaning in `numpy`: https://numpy.org/doc/stable/reference/maskedarray.html And I think it's good for us to stay close to `numpy` conventions as closely as possible.